ros/api/v0/hosts.py

```diff
@@ -26,9 +26,14 @@
 
 class HostsApi(Resource):
     display_performance_score_fields = {
-        'cpu_score': fields.Integer,
-        'memory_score': fields.Integer,
-        'io_score': fields.Integer
+        'cpu': fields.Integer,
+        'memory': fields.Integer,
+        'io': fields.Integer
+    }
```

Comment:
@kgaikwad Do we still need `display_performance_score_fields`?

Reply:
Yes, as mentioned offline while implementation frontend will need these score values to decide colour code.
If we want to remove this field then frontend needs to compute score to apply colour codes per row.

@apuntamb, I think, this field will get removed later based on upcoming colour code changes.

Reply:
Gotcha, sure thing!